RecoEgamma/EgammaElectronProducers/plugins/TrackingRegionsFromSuperClustersProducer.h

```diff
@@ -272,4 +298,15 @@ createTrackingRegion(const reco::SuperCluster& superCluster,const GlobalPoint& v
 							   measTrackerEvent);
 }
 
+void TrackingRegionsFromSuperClustersProducer::validateConfigSettings()const
+{
+  int nrSetTrue=0;
+  if(useZInVertex_) nrSetTrue++;
+  if(useZInBeamspot_) nrSetTrue++;
+  if(useDefaultZ_) nrSetTrue++;
+  if(nrSetTrue!=1){
+    throw cms::Exception("InvalidConfiguration") <<" when constructing TrackingRegionsFromSuperClustersProducer there much be exactly one of useZInVertex(="<<useZInVertex_<<") useZInBeampot(="<<useZInBeamspot_<<") useDefaultZ(="<<useDefaultZ_<<") set";
```

Comment:
(Very minor, but in case you touch this PR you can fix "much" -> "must" in the text...